dtests/src/test/scala/io/snappydata/hydra/ct/CTTestUtil.scala

```diff
@@ -232,15 +234,19 @@ object CTTestUtil {
    */
   def createAndLoadSparkTables(sqlContext: SQLContext, insertUniqueRecords: Boolean): Unit = {
     if (insertUniqueRecords) {
-      CTQueries.orders_details_df(sqlContext).dropDuplicates("SINGLE_ORDER_DID").createOrReplaceTempView("orders_details")
+      CTQueries.orders_details_df(sqlContext, true).dropDuplicates("SINGLE_ORDER_DID")
```

Comment:
Please remove changes in if loop. I have modified this code for fixing a test issue with concurrent putInto.